jsx/doser/calibrate.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import * as Yup from 'yup'
+import { ErrorFor, ShowError } from '../utils/validation_helper'
+import classNames from 'classnames'
+import { withFormik, Field } from 'formik'
+
+export const Calibrate = ({
```

Comment:
Why not using the class pattern for component like everywhere in the UI? Makes it more consistent.

Reply:
It’s ok to have mixed style if we don’t have any state based logic, the react style guide recommends this as well. I remember asking the same to @Ranthalion

Reply:
In react, it is preferred to use stateless function if the component does not need to track state.  Some linters even include this as a rule, but the one we are using doesn't seem to include it.

Reply:
So do you suggest me to migrate in this pattern for Notification Alerts?

Reply:
I don't think it's a big deal.  I quickly looked for any benchmarks that show performance differences, but didn't really find anything compelling.  
Functional components vs Stateful components was a topic when I went to a Reactjs meetup.